README.md

```diff
@@ -124,7 +124,7 @@ good installation path if you are just testing or doing development on the plugi
 
 For more permanent installs of the nexus-repository-composer plugin, follow these instructions:
 
-* Copy the bundle (nexus-repository-composer-0.0.2.jar) into <nexus_dir>/deploy
+* Copy the bundle (nexus-repository-composer-0.0.8.jar) into <nexus_dir>/deploy
```

Comment:
In the past I've resorted to pseudo versions (to avoid stale docs), e.g. `nexus-repository-composer-0.0.X.jar`

You call on whether to do that or not. I reiterate my offer to buy beverages if you find a way to make the markdown handle this automagically.